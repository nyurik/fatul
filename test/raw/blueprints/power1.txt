0eNqNkVFqwzAMQO+ib6c0brImvkopJUlFJrDlYDtjIfjutVMIYyvbQB+SkN4DaYVezzg54gBqBRose1CXFTyN3OncC8uEoIACGhDAnclVT2OBGofgaCgmqxGiAOI7foIq41UAcqBA+IRtxXLj2fTo0sAvGAGT9WnTcnYnWiHbo4AlJ8fkYKTxvbezy2QpTtcofvDlzvem0/pvQ1Md6qejbA71d0v5ynHaHQbvNJt/SNpdUr2UpKttR1ZffiLgA53fMLIpq3Mrz/VbiqqJ8QHQ4ZEQ
