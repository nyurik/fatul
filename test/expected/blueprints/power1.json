{
  "blueprint": {
    "icons": [
      {
        "signal": {
          "type": "item",
          "name": "big-electric-pole"
        },
        "index": 1
      }
    ],
    "entities": [
      {
        "entity_number": 1,
        "name": "big-electric-pole",
        "position": {
          "x": -290,
          "y": -20
        },
        "neighbours": [
          2,
          3
        ]
      },
      {
        "entity_number": 2,
        "name": "small-electric-pole",
        "position": {
          "x": -284.5,
          "y": -18.5
        },
        "neighbours": [
          1
        ]
      },
      {
        "entity_number": 3,
        "name": "medium-electric-pole",
        "position": {
          "x": -289.5,
          "y": -14.5
        },
        "neighbours": [
          1
        ]
      }
    ],
    "item": "blueprint",
    "version": 281479275675648
  }
}
